cli/cppcheckexecutor.cpp

```diff
@@ -1176,9 +1176,9 @@ bool CppCheckExecutor::tryLoadLibrary(Library& destination, const char* basepath
  * Execute a shell command and read the output from it. Returns true if command terminated successfully.
  */
 // cppcheck-suppress passedByValue - "exe" copy needed in _WIN32 code
-bool CppCheckExecutor::executeCommand(std::string exe, std::vector<std::string> args, const std::string &redirect, std::string *output)
+bool CppCheckExecutor::executeCommand(std::string exe, std::vector<std::string> args, const std::string &redirect, std::string *output_)
```

Comment:
I don't understand why we have to change this name? can you enlighten me?

Reply:
There is an object called "output" in the global namespace. Maybe that should be renamed instead, since it's so generic, but this could require more changes.